lambda/cuboid_import_lambda.py

```diff
@@ -133,9 +134,12 @@ def s3_delete(bucket, key, region):
         key (str): S3 object key.
         region (str): AWS region.
     """
-    s3 = boto3.resource('s3', region_name=region)
-    obj = s3.Object(bucket, key)
-    obj.delete()
+    s3 = boto3.client('s3', region_name=region)
+    s3.put_object_tagging(
+        Bucket=bucket, Key=key, Tagging={
```

Comment:
If we try and tag the bucket it fails due to metadata full it will throw and exception that we will be able to search for it later.  In that case, it will be deleted in 21 days instead of 1 but wont effect the impotence of the system. So  if this occurs will see it from the traceback searches and can look at removing more metadata.

Reply:
Actually now I am wondering if throwing an exception is the right thing to do.  We may want to catch such exception and log an error message.  Do we want this to fail if we can't add metadata?

Reply:
Tagging isn't the same as metadata. I don't think this should fail unless S3 isn't available or if tagging permissions aren't granted. But what you're asking is should we just consider the lambda successful as long as the copy succeeded?

Reply:
https://stackoverflow.com/questions/42126348/difference-between-object-tags-and-object-metadata
Your right about the tagging.  There shouldn't be a limit on it.  I am not worried about the tagging causing errors, it was metadata I was worried about.  Your right that shouldn't happen or if it does there are bigger problems happening and we would want to know about them.